homepage/models.py

```diff
@@ -61,6 +90,17 @@ class FollowedUserCourses(models.Model):
     def __str__(self):
         return f'user = {self.user}, course = {self.course}'
 
+    @staticmethod
+    def get_courses_followed_by_appUser(appUser):
+        allFollowedUserCourses = FollowedUserCourses.objects.all()

```

Comment:
Same here, maybe it is possible to use a query to retrieve courses followed by the user, instead of a ```for``` loop.